temporian/core/serialize.py

```diff
@@ -33,7 +33,7 @@ def save(
     outputs: processor.MultipleEventArg,
     path: str,
 ) -> None:
-    """Saves the computation between "inputs" and "outputs" into a file.
+    """Saves the processor between `inputs` and `outputs` to a file.

```

Comment:
suggestion: reword to "containing a set of `inputs`, `outputs`, and `operators`" instead of "between `inputs` and `outputs`", as the `Processor` object contains all of these objects